### PR TITLE
Ubuntu/jammy

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,10 +1,11 @@
-cloud-init (22.3-3-g9f0efc47-0ubuntu1~22.04.1) UNRELEASED; urgency=medium
+cloud-init (22.3-3-g9f0efc47-0ubuntu1~22.04.1) jammy; urgency=medium
```
Comment (blackboxsw) [inline debian/changelog:1]: Not sure why this specifcally is showing up in the visual github visual diff. I don't see it on the cmdline against upstream/ubuntu/jammy. Disregarding.

**Review by blackboxsw — CHANGES_REQUESTED** [review]
- build-package/sbuild success no new lintian warnings
- diff against what was hotfix-released to pkg/ubuntu/jammy-proposed shows all hot-fixed packaging related changes included in this PR.
- FAIL changes to debian/changelog are not recording the "dropped" cherry pick due to new-upstream-snapshot

I think  the following diff is needed and otherwise +1 on this branch:
```diff
diff --git a/debian/changelog b/debian/changelog
index efa2632a..64f3c0d9 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -11,6 +11,8 @@ cloud-init (22.3-3-g9f0efc47-0ubuntu1~22.04.1) jammy; urgency=medium
       https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1014584
   * d/p/expire-on-hashed-users.patch:
     Add patch to ensure password expire doesn't apply to hashed users
+  * drop the following cherry-picks now included:
+    + cpick-a2e62738-Fix-cc_phone_home-requiring-tries-1500
   * New upstream snapshot. (LP: #1987318)
     + DataSourceVMware: fix var use before init (#1674) [Andrew Kutz]
     + rpm/copr: ensure RPM represents new clean.d dir artifacts (#1680)
```

File: debian/changelog (hunk 1 above)
Comment (TheRealFalcon) [reply debian/changelog:1]: I reordered this commit to sit on top of the other two I just added. Strange that github doesn't see that though.

**Comment by blackboxsw** [comment]
Supplemental diff looks good when comparing against what was released into ubuntu/jammy-proposed (from git-ubuntu clone cloud-init; cd cloud-init; git checkout pkg/ubuntu/jammy-proposed). It'll make this easier for SRU reviewers and consumers to understand what cloud-init has officially released.

https://paste.ubuntu.com/p/sRgddwSsTP/

**Review by blackboxsw — APPROVED** [review]

**Pull request merged**